src/components/common/SessionApprove.vue

```diff
@@ -20,9 +20,11 @@
         From
         <Bech32 :address="senderAddress" />
       </div>
-
-      <!-- <TableInvoice :fees="fees" /> -->
-
+      <TableInvoice
+        :amount="amount"
```

Comment:
This won't (yet) pass the tests since the `TableInvoice` code is in the other PR #2802